docs/metrics/confusion-matrix.md

```diff
@@ -0,0 +1,203 @@
+# Confusion Matrix
+
+!!! info inline end "Guide: True Positive / False Positive / False Negative / True Negative"
+
+    You can find more info on true positive, false positive, false negative, true negative in [TP / FP / FN / TN](./tp-fp-fn-tn.md) guide.
```

Comment:
true positive, false positive, false negative,` or` true negative in` the` [] guide.